.golangci.yml

```diff
@@ -1,97 +1,106 @@
+version: "2"
```

Comment:
This was updated locally by running `golangci-lint migrate`, reapplying our existing comments, and doing a sweep through https://golangci-lint.run/product/migration-guide/ to ensure that the changes should be equivalent.